site/content/en/docs/concepts/_index.md

```diff
@@ -51,6 +51,10 @@ A mechanism allowing internal or external components to influence the timing of
 A mechanism allowing to schedule Workloads optimizing Pod placement for
 network throughput between the Pods.
 
+### [Fair Sharing](/docs/concepts/fair_sharing)
+
+Mechanisms in Kueue to share quota between tenants fairly.
+
```

Comment:
Here is API secsion. So could you move this to `Glosaray` section like `Preemption`?

Reply:
Done